tests/JustSaying.TestingFramework/AwsFactAttribute.cs

```diff
@@ -6,11 +6,6 @@ namespace JustSaying.TestingFramework;
 public sealed class AwsFactAttribute : FactAttribute
 {
     public AwsFactAttribute()
```

Comment:
If credentials are redundant, we can just remove the need for these two derived attributes at all?